sdk/src/main/java/com/karhoo/sdk/api/KarhooSDKConfiguration.kt

```diff
@@ -13,4 +14,6 @@ interface KarhooSDKConfiguration {
     fun authenticationMethod(): AuthenticationMethod
 
     fun analyticsProvider(): AnalyticProvider?
+
+    suspend fun requestNewAuthenticationCredentials(callback: suspend (Credentials) -> Unit)
```

Comment:
The DP cannot pass `Credentials` because they don't have access to them, but we have and can retrieve them ourselves as long as they re-login and call the callback.